packages/roosterjs-editor-api/lib/format/insertTable.ts

```diff
@@ -12,6 +21,14 @@ export interface TableFormat {
     borderCollapse?: string;
 }
 
+/**
+ * Insert table into editor at current selection
+ * @param editor The editor instance
+ * @param columns Number of columns in table, it also controls the default table cell width:
+ * if columns <= 4, width = 120px; if columns <= 6, width = 100px; else width = 70px
+ * @param rows Number of rows in table
+ * @param format (Optional) The table format
```

Comment:
If this is null, what's the default format

Reply:
Added null case